timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -273,21 +275,28 @@ export class TimeGraphChart extends TimeGraphChartLayer {
                 this.updateZoomingSelection();
             }
         };
-        this.onCanvasEvent('mousedown', mouseDownListener);
+        this.onCanvasEvent('mousemove', this._mouseMoveHandler);
+        this.onCanvasEvent('keydown', this._keyDownHandler);
+        this.onCanvasEvent('keyup', this._keyUpHandler);
+        this.onCanvasEvent('mousedown', this._mouseDownHandler);
+        this.onCanvasEvent('mousewheel', this._mouseWheelHandler);
+        this.onCanvasEvent('wheel', this._mouseWheelHandler);
+        this.onCanvasEvent('contextmenu', this._contextMenuHandler);
 
         this.rowController.onVerticalOffsetChangedHandler(verticalOffset => {
             this.layer.position.y = -verticalOffset;
         });
 
-        this.unitController.onViewRangeChanged(() => {
+        this._viewRangeChangedHandler = () => {
```

Comment:
This handler doesn't get removed and still gets called after time graph is closed.

Reply:
Now it gets removed with https://github.com/theia-ide/theia-trace-extension/pull/342.